src/Vsix/Microsoft.VisualStudio.InteractiveWindow.vsmanproj

```diff
@@ -1,23 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
   <Import Project="..\..\build\Targets\Settings.targets" />
 
   <PropertyGroup>
     <FinalizeManifest>true</FinalizeManifest>
     <FinalizeSkipLayout>true</FinalizeSkipLayout>
     <IsPackage>true</IsPackage>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(RunningInMicroBuild)' != 'true'">
+    <FinalizeValidate>false</FinalizeValidate>
+    <ManifestPublishUrl>https://vsdrop.corp.microsoft.com/file/v1/Products/DevDiv/dotnet/testimpact/dev/$(BuildNumber)</ManifestPublishUrl>
```

Comment:
You probably want a different URL for you unsigned builds (this one says testimpact).